prettier.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
This file should be in the gitignore since it's auto generated.

Reply:
added to .gitignore